sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/TypeFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.mercedesbenz.sechub.commons.model;
+
+public interface TypeFilter {
+    public static class AcceptAllTypeFilter implements TypeFilter {
+
+        @Override
+        public boolean isTypeAccepted(String type) {
+            return true;
+        }
+    }
+
+    public static class AcceptTypeFilter implements TypeFilter {
+
+        private String type;
+
+        public AcceptTypeFilter(String type) {
+            this.type = type;
+        }
+
+        @Override
+        public boolean isTypeAccepted(String type) {
+            if (type == null) {
+                return false;
+            }
+            // more than one type can be defined eg. docker,git,binary
+            final String[] splitTypes = type.split(",");
+            for (String splitType : splitTypes) {
+                if (this.type.equals(splitType)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+    }
+
+    public static final AcceptAllTypeFilter ACCEPT_ALL = new AcceptAllTypeFilter();
```

Comment:
Please move static parts always on top of main class - here at the interface.

We have always following order

1. fields
2. methods
3. internal classes

And to separated this again:
1. static parts before "normal" parts
4. constructors before other methods
5. public before protected, before package private, before private